validator/validator-main.protoascii

```diff
@@ -5003,6 +5003,7 @@ attr_lists: {
   attrs: { name: "aria-labelledby" }
   attrs: { name: "aria-level" }
   attrs: { name: "aria-live" }
+  attrs: { name: "aria-modal" }
```

Comment:
perhaps add `value: ""` so that it is only present as `aria-model` with no possible values.

Reply:
I'm not sure because I think it can have the value of `true` or `false`, optionally. Other ARIA attributes are the same, like `aria-multiline` and `aria-multiselectable`, etc.

Reply:
The code that alerted me to this attribute being missing is [found](https://github.com/WordPress/wordpress-develop/blob/0cab817ba4ec57224c3e7048dbe57478851ca813/src/wp-includes/blocks/navigation.php#L534) in WordPress core and it has a value:

```html
<div class="wp-block-navigation__responsive-dialog" role="dialog" aria-modal="true" aria-labelledby="modal-%1$s-title" >
```

Reply:
Then lets leave it as is. Thanks for checking!